### DISCUSSION TITLE
Routes listing displays routes multiple times

### DISCUSSION THREAD

**Discussion by bamyers99** [transitland/transitland#384] [Q&A]
https://www.transit.land/operators/o-dr5r-nyct#routes clicking "load more" displays routes multiple times.
For example, route L0719 is listed 4 times.

**Comment by irees** [comment]
hi @bamyers99 

Thanks for noticing this. 

In this particular case, it's because the MTA NYCT operator aggregates data from several different GTFS feeds published by the MTA. MTA publishes a separate static feed for the subway, each borough's bus service, LIRR, etc. The routes and stops are duplicated in each of the component feeds, so it's included multiple times in the output when searching by the operator. The MTA situation is a little unique; only a couple other operators in our system split the data into multiple feeds and also duplicate routes/stops between them. We'll look into deduplicating these from the output.
